src/linux/Packaging.Linux/build.sh

```diff
@@ -79,7 +121,7 @@ if [ $INSTALL_FROM_SOURCE = true ]; then
     echo "Install complete."
 else
     # Pack
-    "$INSTALLER_SRC/pack.sh" --configuration="$CONFIGURATION" --payload="$PAYLOAD" --symbols="$SYMBOLS" --version="$VERSION" || exit 1
+    "$INSTALLER_SRC/pack.sh" --configuration="$CONFIGURATION" --arch="$ARCH" --payload="$PAYLOAD" --symbols="$SYMBOLS" --version="$VERSION" || exit 1
```

Comment:
Let's keep the .NET runtime identifier where we can, and only use the 'arch' for the debian package control file.

```suggestion
    "$INSTALLER_SRC/pack.sh" --configuration="$CONFIGURATION" --runtime="$RUNTIME" --payload="$PAYLOAD" --symbols="$SYMBOLS" --version="$VERSION" || exit 1
```